Cargo.toml

```diff
@@ -1,6 +1,8 @@
 [workspace]
-
 members = [
   "simplefs",
-  "simplefs-fuse"
+  "simplefs-fuse",
+
+  # Private crates
+  "examples",
```

Comment:
is it necessary to make examples its own crate?

Reply:
As opposed to what?

Reply:
I'll see if I can do what i was originally trying to do and PR it to here

Reply:
👍